pkgs/applications/networking/instant-messengers/riot/riot-desktop.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv, fetchFromGitHub, yarn2nix, makeDesktopItem, electron, riot-web }:
+
+let
+  executableName = "riot-desktop";
```

Comment:
If supports `pname` use that throughout instead of this.

Reply:
`pname` isn't supported. Keeping as is.